README.md

```diff
@@ -11,179 +10,188 @@ for applicable air flight routes from a previously populated database. An additi
 
 ![Application](app.png)
 
+
 ## Prerequisites
 
 To download the application you can either download [the archive](https://github.com/couchbaselabs/try-cb-nodejs/archive/master.zip) or clone the repository:
 
-```
-git clone https://github.com/couchbaselabs/try-cb-nodejs.git
-```
+    ```
```

Comment:
Looks like the indentation on the block doesn't work, probably it was because this was a in bullet points previously, so it worked then.

Reply:
hmm, so we could use *just* indentation, or the triple-backtick github extension...
that said, https://github.github.com/gfm/#fenced-code-blocks does say 

> Fences can be indented. If the opening fence is indented, content lines will have equivalent opening indentation removed, if present

so not sure why it's not stripping the fencing

Reply:
@simon-dew notes that fenced blocks can only be indented 0-3 characters. At 4, the "this is an indented block" behaviour kicks in.